src/service.rs

```diff
@@ -563,58 +544,22 @@ where
 
     /// Spawn protocol handle
     #[inline]
-    fn handle_open(
-        &mut self,
-        handle: InnerProtocolHandle,
```

Comment:
What’s the purpose of using `InnerProtocolHandle` and why it can be removed now. Especially, why `InnerProtocolHandle::Service` has gone?

Reply:
There is no need to force the unification of the two handle opening procedures, which adds complexity.

The service level only needs to be opened once globally, with no subsequent scenarios opened at all, while the session-level only needs to iterate through all current possibilities at once.